extensions/amp-story-auto-ads/0.1/test/test-story-ad-config.js

```diff
@@ -204,4 +204,51 @@ describes.realWin('amp-story-auto-ads:config', {amp: true}, (env) => {
       });
     });
   });
+
+  it('does use remote config when src attribute is provided', async () => {
```

Comment:
Can we add a few more tests for invalid remote config, bad url, or any other common failure case? Right now I think we only cover the good case but we also want to test that we give helpful errors when a user misconfigures.